### ISSUE TITLE
frontend image upload returns 500 error

### ISSUE THREAD

**Issue by dylanbannon** [vanvalenlab/kiosk#29]
I've observed this on GKE when starting up a cluster, destroying it, and restarting the cluster, all without restarting the kiosk.

**Comment by willgraf** [comment]
Idea: The cluster could not properly delete the bucket during destruction.

**Comment by dylanbannon** [comment]
`kubectl logs frontend-deployment-[blahblah]-[blah]` shows this:
```
2018-11-12T17:02:48.276Z [ERROR] deepcell-20181111-1@deepcell-209717.iam.gserviceaccount.com does not have storage.objects.delete access to deepcell-output/uploads/img_000000000_FITC_001.png.
```

**Comment by willgraf** [comment]
Just saw this myself, deploying on GKE:
```
2018-11-12T19:51:13.619Z [ERROR] training-test@deepcell-209717.iam.gserviceaccount.com does not have storage.objects.delete access to deepcell-output/uploads/mibi_nuclear.png.
Mon, 12 Nov 2018 19:51:13 GMT express:router <anonymous>  : /api/upload
2018-11-12T19:51:13.620Z [ERROR] middlewareError
Mon, 12 Nov 2018 19:51:13 GMT express:router <anonymous>  : /api/upload
2018-11-12T19:51:13.622Z [INFO] ::ffff:10.24.2.7 - - "POST /api/upload HTTP/1.1" 500 148 "http://104.196.231.104/predict" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"

RangeError: Invalid status code: undefined
    at ServerResponse.writeHead (_http_server.js:199:11)
    at ServerResponse.writeHead (/usr/src/app/node_modules/on-headers/index.js:55:19)
    at ServerResponse.writeHead (/usr/src/app/node_modules/on-headers/index.js:55:19)
    at ServerResponse._implicitHeader (_http_server.js:190:8)
    at ServerResponse.end (/usr/src/app/node_modules/compression/index.js:103:14)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:221:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:267:15)
    at app.use (/usr/src/app/dist/server/config/express.js:95:26)
    at Layer.handle_error (/usr/src/app/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:315:13)
```

**Comment by willgraf** [comment]
Deploying on AWS, I get a 500 error for all upload requests, even those that are new files.

**Comment by dylanbannon** [comment]
Yeah, @willgraf, I'm not seeing this on AWS, so I'm going to move forward with the understanding that this is a GKE-only issue.

**Comment by dylanbannon** [comment]
Fixed by #75. See there for full explanation of bug.
